content/en/docs/configuration/external/aws-privateca-issuer/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "AWS Private CA Issuer"
+linkTitle: "AWS Private CA Issuer"
+weight: 60
+type: "docs"
+---
+
+Kubernetes containers and applications use digital certificates to provide secure authentication and encryption over TLS. A widely adopted solution for TLS certificate life-cycle management in Kubernetes is cert-manager, an add-on to Kubernetes that requests certificates, distributes them to Kubernetes containers, and automates certificate renewal.
```

Comment:
Suggestion: I think this line can be removed since as the reader is already on cert-manager website
```suggestion

```

Reply:
No action. This documentation will be available on AWS tech docs via a link. It needs to provide context for multiple audiences.